functionary/VSCODE.md

```diff
@@ -0,0 +1,72 @@
+# Developing using VSCode
+
+For users of VSCode, a variety of tooling is provided in the functionary
+repository to help ease the development workflow.
+
+## Workspaces
+
+For organization purposes, treat each directory at the root of the repository as
+its own workspace. That is, rather than opening the root folder in VSCode, open
+one of the folders at that root level. For example:
+
+- functionary
+- runner
+- cli
```

Comment:
Yeah, this is probably worth playing around with.  I tried out the multi-folder workspace thing at some point and didn't love it, but I can't remember what about it bothered me and that was prior to this specific effort.  I'll revisit that and see if I can get it to a place I'm happy with because I agree that it would simplify the workflow a bit.

Reply:
I created a functionary.code-workspace file and reorganized things accordingly.  I think that streamlines things nicely.

I only added the functionary, runner, and cli folders to the workspace.  Basically the things that you might theoretically debug (though there's no launch setup for the cli currently).